src/VkLayer_GF_amber_scoop/src/draw_call_tracker.cc

```diff
@@ -14,17 +14,34 @@
 
 #include "VkLayer_GF_amber_scoop/draw_call_tracker.h"
 
+#include <vulkan/vulkan.h>
+
 #include <atomic>
-#include <cstdint>
 #include <fstream>
+#include <memory>
 #include <sstream>
+#include <stdexcept>
 #include <string>
 
 #include "VkLayer_GF_amber_scoop/amber_scoop_layer.h"
+#include "VkLayer_GF_amber_scoop/create_info_wrapper.h"
+#include "VkLayer_GF_amber_scoop/graphics_pipeline_data.h"
 #include "gf_layers_layer_util/logging.h"
+#include "gf_layers_layer_util/spirv.h"
+#include "gf_layers_layer_util/util.h"
+
+#pragma warning(push, 1)  // MSVC: reduces warning level to W1.
+
+#include "spirv-tools/libspirv.h"
+#include "spirv-tools/libspirv.hpp"
+
+#pragma warning(pop)
 
 namespace gf_layers::amber_scoop_layer {
 
+static std::string DisassembleShaderModule(
+    const VkShaderModuleCreateInfo& create_info);
```

Comment:
I have moved this function up to avoid the forward declaration and used an anonymous namespace instead of using `static`.